ginga/rv/main.py

```diff
@@ -231,6 +232,8 @@ def add_default_options(self, argprs):
             # newer ArgParse
             add_argument = argprs.add_argument
 
+        add_argument("--basedir", dest="basedir", metavar="NAME",
```

Comment:
Does this also change where Ginga look for other things like `ginga_config.py` and `*.cfg` files?

Reply:
Yes, so you can have multiple "configurations" for Ginga just by creating different directories and using this option. Each one is completely separate.